packages/mongodb-chatbot-ui/src/App.tsx

```diff
@@ -84,7 +85,10 @@ function App() {
           serverBaseUrl={serverBaseUrl}
           shouldStream={shouldStream}
           darkMode={darkMode}
-          fetchOptions={{ credentials: "include" }}
+          fetchOptions={{
+            credentials: "include",
+            headers: getSegmentIdHeaders(),
```

Comment:
should this also be included in the docs/uni/devcenter implementations of the chatbot?

Reply:
They don't actually use the built in ones anymore - will make separate tickets for each team to integrate into their respective implementations.